src/service/fixed-layer.js

```diff
@@ -557,9 +536,18 @@ export class FixedLayer {
 
     // Update `top`. This is necessary to adjust position to the viewer's
     // paddingTop.
-    if (state.top && (state.fixed || (state.sticky && !this.transfer_))) {
```

Comment:
I believe both `fixed` and `sticky` could be false. Is it valid to remove it from this expression in the new code? It might have some undesired affect inside the `position:relative` parent...